src/folding/nova/circuits.rs

```diff
@@ -690,9 +640,9 @@ pub mod tests {
         let cs = ConstraintSystem::<Fr>::new_ref();
 
         // prepare the circuit to obtain its R1CS
-        let F_circuit = TestFCircuit::<Fr>::new();
+        let F_circuit = CubicFCircuit::<Fr>::new(());
         let mut augmented_F_circuit =
-            AugmentedFCircuit::<Projective, Projective2, GVar2, TestFCircuit<Fr>>::empty(
+            AugmentedFCircuit::<Projective, Projective2, GVar2, CubicFCircuit<Fr>>::empty(
```

Comment:
It's a bit weird to create a `TestCircuit` struct and then use `CubicFCircuit` on them. I'm not sure I follow this. Maybe I'm missing something?

Reply:
So the idea is that `FCircuit` is the interface, and we have 2 existing 'test circuits' which are `CubicFCircuit` and `CustomFCircuit`, both of them implementing the trait `FCircuit` (more details in the doc of each of them 
https://github.com/privacy-scaling-explorations/folding-schemes/tree/55fa83bf66648f1c4fb1709662fbc0b871b4ffc1/src/frontend/mod.rs#L46).

Reply:
Hmmm I see. Makes sense!